core/src/main/java/io/lionweb/lioncore/java/serialization/ProtoBufSerialization.java

```diff
@@ -160,11 +160,13 @@ public byte[] serializeToByteArray(SerializedChunk serializedChunk) {
     return serialize(serializedChunk).toByteArray();
   }
 
-  private class SerializeHelper {
-    final Map<MetaPointer, Integer> metaPointers = new HashMap<>();
-    final Map<String, Integer> strings = new HashMap<>();
+  protected class SerializeHelper {
+    public final Map<MetaPointer, Integer> metaPointers = new HashMap<>();
```

Comment:
`public` sounds suspicious for fields (same for next line)

Reply:
Changed